### PR TITLE
CTM-611: added the logic for batch match run messaging. also fixed CT…

### PR DESCRIPTION
**Pull request by kevinwangcanada**
…M-626 subject line for success run email.

### CODE REVIEW

File: apps/api/src/app/message-queue/message-queue.service.ts [hunk 1]
```diff
@@ -137,6 +142,51 @@ export class MessageQueueService implements OnModuleInit, OnModuleDestroy {
...
+        }
+      });
+      const result2 = await this.prismaService.trial.updateMany({
+        where: {
+          trial_internal_id: {
+            in: failedTrialInternalIds
+          },
+          trial_status: {
+            in: [TrialStatusEnum.PENDING,TrialStatusEnum.MATCHED]
+            }
```
Comment (mickey-ng) [inline apps/api/src/app/message-queue/message-queue.service.ts:166]: when they are sent to matchminer they should all the PENDING, so I don't think MATCHED is needed here

File: apps/api/src/app/message-queue/message-queue.service.ts [hunk 2]
```diff
@@ -137,6 +142,51 @@ export class MessageQueueService implements OnModuleInit, OnModuleDestroy {
 
       await sendMail(from, to, subject, mailTemplate);
     }
+    else if (json_message.run_status === 'partial_success') {
```
Comment (mickey-ng) [inline apps/api/src/app/message-queue/message-queue.service.ts:145]: Can you create an enum class for the run_status?

File: apps/api/src/app/message-queue/message-queue.service.ts (hunk 1 above)
Comment (kevinwangcanada) [reply apps/api/src/app/message-queue/message-queue.service.ts:166]: Yes, I think you are right. I was thinking something else. will fix

File: apps/api/src/app/message-queue/message-queue.service.ts (hunk 2 above)
Comment (kevinwangcanada) [reply apps/api/src/app/message-queue/message-queue.service.ts:145]: yes. will do.

**Review by mickey-ng — APPROVED** [review]

**Pull request merged**